tests/storage/misc_test.py

```diff
@@ -576,8 +576,9 @@ def testStdOut(self):
         """
         line = ("All I wanted was to have some pizza, hang out with dad, "
                 "and not let your weirdness mess up my day")
-        # (C) Nickolodeon - Invader Zim
+        # (C) Nickelodeon - Invader Zim
```

Comment:
This change is not related. The real fix is to remove the copyrighted text from the tests, instead of fixing the copyright. The tests license is GPL and they cannot contain text with another copyright.